### ISSUE TITLE
add_to_watchlist fails

### ISSUE THREAD

**Issue by PeterEnberg** [Qluxzz/avanza#115]
add_to_watchlist has worked well but has regressed during the last month or so.

```
http.client.RemoteDisconnected: Remote end closed connection without response
```
From manual web interaction I see that that the URL does not match what the API is trying to use. It has changed to:

`/_api/watchlist/watchlist/add/<watchlist_id>/<orderbook_id>`

Remove from watchlist has also changed and is now using:

`/_api/watchlist/watchlist/remove/<watchlist_id>/<orderbook_id>`

Furthermore both calls now use **POST** instead of PUT and DELETE respectively.

I will submit a PR.

**Comment by PeterEnberg** [comment]
Do I need to be granted push access or am I doing something wrong when pushing?

**Comment by Qluxzz** [comment]
These changes are included in the newly released v13.0.0 version
